packages/venia-ui/lib/components/RadioGroup/radio.js

```diff
@@ -13,24 +14,30 @@ export class RadioOption extends Component {
         classes: shape({
             input: string,
             label: string,
-            root: string
+            root: string,
+            icon: string
         }),
         label: node.isRequired,
         value: node.isRequired
     };
 
     render() {
         const { props } = this;
-        const { classes, id, label, value, ...rest } = props;
+        const { classes, id, initialChecked, label, value, ...rest } = props;
 
         return (
             <label className={classes.root} htmlFor={id}>
                 <Radio
                     {...rest}
                     className={classes.input}
                     id={id}
+                    defaultChecked={initialChecked}
```

Comment:
Good catch.

Reply:
A radio should get its checked state from the parent RadioGroup value. Just curious why this was needed.